challenge.md

```diff
@@ -31,19 +31,19 @@ Open up this file in your text editor, and write your answers below (where indic
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
+ <!-- Answer Starts Here --> an absolute path because the path is completely listed out
  
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
- <!-- Answer Starts Here -->
+<!-- Answer Starts Here -->  a absolute path because it is not on my pc
  
- <!-- Answer Ends Here -->
+ <!-- Answer Ends Here --> 
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
- <!-- Answer Starts Here -->
+ <!-- Answer Starts Here --> an absolute path because we would list it completely out.
```

Comment:
No, this is not correct. An Absolute path defines a fixed location - like lat/long GPS coordinates. If we were to bundle up our project in a zip, and share it with a friend, when they opened up the project, any links with Absolute paths would point back to the _old location of the files_, rather than moving with the project. Relative paths, however, will stay accurate even as the project moves, _because the files are all in the same relative locations_. 

As a rule of thumb, use Relative paths to link to resources within your project and use Absolute paths to link to resources outside your project.